### PR TITLE
(mpc-hc-clsid2) AU Fix for Stable Releases

### PR DESCRIPTION
**Pull request by TreyRuffy**
## Description
I have made it so that MPC-HC-CLSID2 will let the AU get the latest stable releases without outputting an error.

## Motivation and Context
This fixes the AU giving an error every time it runs because MPC-HC-CLSID2 does not have conventional update names.

## How Has This Been Tested?
I have run it locally along with a private PR to ensure the bot will run correctly. They both were successful.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/mpc-hc-clsid2/update.ps1 [hunk 1]
```diff
@@ -27,8 +27,7 @@ function global:au_GetLatest {
   $re = 'x86\.exe$'
   $url32 = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
 
-  $re = 'x64\.exe$'
-  $url64 = $download_page.links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
+  $url64 = $url32 -replace "x86","x64"
```
Comment (AdmiringWorm) [inline automatic/mpc-hc-clsid2/update.ps1:30]: Do not rely on blindly replacing `x86` with `x64`.
There have been times in the past where the naming haven't been exact.

TBH, this change looks entirely unnecessary.
Please explain why this change is necessary.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: automatic/mpc-hc-clsid2/update.ps1 (hunk 1 above)
Comment (AdmiringWorm) [reply automatic/mpc-hc-clsid2/update.ps1:30]: I see know why you made that change.
However, my first two lines still stands as the name isn't guaranteed to be the same.

Please only update the regex to also check the version instead.

_Remember to always update the commit message as mentioned in your other message_

File: automatic/mpc-hc-clsid2/update.ps1 [hunk 2]
```diff
@@ -22,24 +22,24 @@ function global:au_SearchReplace {
...
+  $releasesPage = Invoke-WebRequest -Uri $releases -UseBasicParsing
 
-  $re = 'x86\.exe$'
-  $url32 = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
+  $re = "^((?:\d+\.){2}\d+)$"
 
+  $versionHtml = $releasesPage.Links | ? title -match $re | select -first 1
+  $version = $versionHtml.title
+
+  $download_page = Invoke-WebRequest -Uri "$($releases)/tag/$($version)"
```
Comment (RedBaron2) [inline automatic/mpc-hc-clsid2/update.ps1:32]: The time out is being caused by the lack of using the `-UseBasicParsing` switch.

Comment (TreyRuffy) [reply automatic/mpc-hc-clsid2/update.ps1]: Thank you. I just pushed a commit to add that.
